### DISCUSSION TITLE
Screen brightness problem

### DISCUSSION THREAD

**Discussion by angrycsm** [LingmoOS/Community#30] [Bugs Report/缺陷提交] [answered]
Hey guys, good morning. I'm here again to report a problem with my distro.

Well, recently a message has been appearing on my screen asking for sudo permission to change the brightness of my screen and this ends up bothering me a lot of the time.

How do I solve this?

Problem in English:

Authentication is required to run `...` as the superuser

![Screenshot_20241216_055634](https://github.com/user-attachments/assets/9aaf9313-fe1c-4646-bbac-c8f4c602ef8d)

**Comment by samwhelp** [comment, marked as answer]
This issue has been mentioned in [Discussions #25](https://github.com/orgs/LingmoOS/discussions/25), but it is not English.


The simple way to fix this,

edit file: [/etc/profie](https://github.com/samwhelp/lingmo-adjustment/blob/main/sample/demo/profile-config/prepend-path/asset/overlay/etc/profile#L3-L53)

or you can download my /etc/profile sample to fix this

``` sh
wget https://raw.githubusercontent.com/samwhelp/lingmo-adjustment/refs/heads/main/sample/demo/profile-config/prepend-path/asset/overlay/etc/profile
```

then replace /etc/profile

``` sh
sudo install -Dm644 profile /etc/profile
```

**Reply by angrycsm** [reply]
Thank youuu!!
